Documentation/ceph-examples.md

```diff
@@ -19,11 +19,12 @@ See the **[example yaml files](https://github.com/rook/rook/blob/{{ branchName }
 
 ## Common Resources
 
-The first step to deploy Rook is to create the common resources. The configuration for these resources will be the same for most deployments.
-The [common.yaml](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/common.yaml) sets these resources up.
+The first step to deploy Rook is to create the CRDs and other common resources. The configuration for these resources will be the same for most deployments.
+The [crd.yaml](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/crd.yaml) and
+[common.yaml](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/common.yaml) sets these resources up.
 
 ```console
-kubectl create -f common.yaml
+kubectl create -f crd.yaml -f common.yaml
```

Comment:
This is a very minor nit, but we could consider `crds.yaml` instead of `crd.yaml`. I tend to imagine `crds.yaml` in my head. Istio uses `crds.yaml` and a few others do use `crds.yaml`.

Reply:
Agreed, i'll rename it